ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from ideascube.serveradmin.catalog import NoSuchPackage, Catalog
+
+
+class Command(BaseCommand):
+    help = 'Manage apps and content'
+
+    def add_arguments(self, parser):
+        subs = parser.add_subparsers(
+            title='Commands', dest='cmd', metavar='',
+            parser_class=argparse.ArgumentParser)
+        subs.required = True
+
+        required_ids = argparse.ArgumentParser('required_ids', add_help=False)
+        required_ids.add_argument(
+            'ids', nargs='+', metavar='ID', help='One or more package ids')
+
+        # -- Manage content ---------------------------------------------------
+        list = subs.add_parser('list', help='List packages')
```

Comment:
Would be friendly to say "No remotes yet please add one with..." when calling "list" while not having added any remote :)